bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/internal/components/UIComponentSitemapProvider.java

```diff
@@ -99,6 +99,11 @@ public class UIComponentSitemapProvider implements SitemapProvider, RegistryChan
 
     private static final Pattern CONDITION_PATTERN = Pattern
             .compile("(?<item>[A-Za-z]\\w*)?\\s*(?<condition>==|!=|<=|>=|<|>)?\\s*(?<sign>\\+|-)?(?<state>.+)");
+    private static final Pattern COMMANDS_PATTERN1 = Pattern.compile("^\"(?<cmd1>[^\"]*)\":\"(?<cmd2>[^\"]*)\"$");
```

Comment:
I think this single regex will do:
`"^(?<cmd1>\"[^\"]*\"|[^\: ]*):(?<cmd2>.*)$"`
You just need to strip the opening and closing double quotes from `cmd1` and `cmd2` after the match.